db/schema.rb

```diff
@@ -19,8 +19,28 @@
     t.string "website"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.text "introduction"
```

Comment:
Can this be removed? I think it's a leftover from your local development database. Removing it manually from `db/schema.rb` is fine.